crates/starknet/src/starknet/mod.rs

```diff
@@ -629,7 +629,12 @@ impl Starknet {
     }
 
     /// Creates an invoke tx for minting, using the chargeable account.
-    pub async fn mint(&mut self, address: ContractAddress, amount: u128) -> DevnetResult<Felt> {
+    pub async fn mint(
+        &mut self,
+        address: ContractAddress,
+        amount: u128,
+        erc20_contract: &str,
```

Comment:
Can the name reflect the fact that it's an address abstraction, not a contract one?